src/main/java/uk/gov/companieshouse/config/SecurityConfig.java

```diff
@@ -4,30 +4,39 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import uk.gov.companieshouse.api.interceptor.InternalUserInterceptor;
 import uk.gov.companieshouse.api.interceptor.TokenPermissionsInterceptor;
 import uk.gov.companieshouse.interceptor.DissolutionTokenPermissionsInterceptor;
 
 @Configuration
 public class SecurityConfig implements WebMvcConfigurer {
 
-    private static final String URI_PATTERN = "/dissolution-request/*";
+    private static final String URI_PATTERN = "/dissolution-request/**";
```

Comment:
`/dissolution-request/*` will check endpoints within one path after `/dissolution-request/` (i.e., before the next "/"), such as `/dissolution-request/{company-number}`, but not `/dissolution-request/{company-number}/payment`.
So this should be `/dissolution-request/**` so anything after `/dissolution-request/` is checked.